systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/AdminClientOperations.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources.crd.kafkaclients;
+
+public enum AdminClientOperations {
```

Comment:
Similarly to the comment below ... this seems a very single-purpose thing with a very generic name. Are these only Topic operations? Does it have ambitions to contain all operations int he future? What is operation in the first place?

Reply:
Probably same story like in comment above/below. For now only topic operations, and we can enhance it later when needed. 
Should I rename it to `AdminClientTopicOperations`? 
Or rather rename enum operations with suffixing them with `_TOPIC`? (`CREATE_TOPIC`, `DELETE_TOPIC`, ...)

Reply:
We don't know ATM how we will use this AdminClient in the future ... so maybe renaming it to `AdminClientTopicOperations` will be better. @Frawless what do you think?

Reply:
Already renamed enum like this, but I have no problem to change it again :-)
```
public enum AdminClientOperations {
    CREATE_TOPICS("create"),
    DELETE_TOPICS("delete"),
    LIST_TOPICS("list"),
    UPDATE_TOPICS("update"),
    HELP("help");
```